homeassistant/components/zwave_js/device_trigger.py

```diff
@@ -249,7 +247,7 @@ def get_trigger_platform_from_type(trigger_type: str) -> str:
 
 async def async_get_triggers(
     hass: HomeAssistant, device_id: str
-) -> list[dict[str, Any]]:
+) -> list[dict[str, str]]:
```

Comment:
This doesn't match what the type of the values in the dict in in the return value may be here.

Is there something that says it should only be strings?

Reply:
I'll double-check

Reply:
I have reviewed this, and I have adjusted the pylint plugin accordingly: see #72126

Reply:
I have reviewed this, and I have adjusted the pylint plugin accordingly: see #72126